bench/tach.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Polymer/tachometer/master/config.schema.json",
+  "timeout": 0.001,
+  "sampleSize": 25,
```

Comment:
is this lower than the default?

Reply:
Yes, so far the results have been clear and haven't needed long timeouts.